pkg/agent/util/net_linux.go

```diff
@@ -453,6 +454,12 @@ func PrepareHostInterfaceConnection(
 		return "", false, fmt.Errorf("failed to set link up: %v", err)
 	}
 
+	if mtu > 0 {
+		if err := bridge.SetInterfaceMTU(ifaceName, mtu); err != nil {
+			return "", false, fmt.Errorf("failed to set bridge interface MTU: %w", err)
+		}
+	}
```

Comment:
This is not enforced if the port already exists (as determined by the call to `bridge.GetOFPort` at the beginning of the function). My take is that this is consistent with how we handle other configurations at the moment (e.g., we don't call `SetLinkUp` unconditionally).